pkg/crc/validation/validation.go

```diff
@@ -86,6 +75,20 @@ func ValidateBundle(bundlePath string, preset crcpreset.Preset) error {
 	return nil
 }
 
+func bundleMismatchWarning(userProvidedBundle string, preset crcpreset.Preset) {
+	userProvidedBundle = bundle.GetBundleNameWithExtension(userProvidedBundle)
+	if userProvidedBundle != constants.GetDefaultBundle(preset) {
+		// Should append underscore (_) here, as we don't want crc_libvirt_4.7.15.crcbundle
+		// to be detected as a custom bundle for crc_libvirt_4.7.1.crcbundle
+		usingCustomBundle := strings.HasPrefix(bundle.GetBundleNameWithoutExtension(userProvidedBundle),
+			fmt.Sprintf("%s_", bundle.GetBundleNameWithoutExtension(constants.GetDefaultBundle(preset))))
+		if usingCustomBundle {
+			logging.Warnf("Using custom bundle %s", userProvidedBundle)
```

Comment:
Oh, I hadn't paid attention until now, but by removing the early `return` which was present before `Validation: add bundleMismatchWarning helper to warn user`, you need to add ` Validation: separate custom bundle and mismatch bundle warning` later on. Would be worth squashing  both together.